hedera-node/src/main/java/com/hedera/services/queries/contract/GetContractInfoAnswer.java

```diff
@@ -151,7 +157,7 @@ private void setAnswerOnly(
 				response.setContractInfo((ContractGetInfoResponse.ContractInfo) ctx.get(CONTRACT_INFO_CTX_KEY));
 			}
 		} else {
-			var info = view.infoForContract(op.getContractID(), aliasManager);
+			var info = view.infoForContract(op.getContractID(), aliasManager, dynamicProperties.maxTokensPerAccount());
```

Comment:
It might be better to rename the property to mention `....ForAccountInfo` ( `maxTokensForAccountInfo` ?) in it for clarity